src/pages/patientView/clinicalInformation/ClinicalInformationSamplesTable.tsx

```diff
@@ -12,48 +13,69 @@ type TODO = any;
 
 export class ClinicalInformationSamplesTable extends React.Component<IClinicalInformationSamplesTableProps, any> {
 
-    render() {
+    public render() {
 
-        const sampleTableData = convertSamplesData(this.props.samples);
+        const sampleInvertedData: TODO = convertSamplesData(this.props.samples);
 
-        const headerCells = sampleTableData.columns.map((col: TODO, i: number) => {
-            return (<th style={{ whiteSpace: 'nowrap' }} key={i}>
-                       <SampleLabelHTML color={'black'} label={(i + 1).toString()} />
-                       {' ' + col.id}
-                   </th>);
-        });
+        const tableData = this.prepareData(sampleInvertedData);
 
-        const rows: Array<JSX.Element> = [];
-
-        Object.keys(sampleTableData.items).forEach((key: string) => {
-            const rowData: TODO = sampleTableData.items[key];
-            rows.push(
-                <tr key={key}>
-                    <td key={-1}>{rowData.clinicalAttribute.displayName}</td>
-                    {
-                        sampleTableData.columns.map((col: TODO, i: number) => {
-                            if (col.id in rowData) {
-                                return <td key={i}>{rowData[col.id]}</td>;
-                            } else {
-                                return <td key={i}>N/A</td>;
-                            }
-                        })
-                    }
-                </tr>
-            );
-        });
+        const headerCells = this.buildHeaderCells(sampleInvertedData);
 
         return (
-            // undo global css styles from cbioportal
-            <Table striped style={{borderCollapse: 'unset', borderSpacing: '0px'}}>
-                <thead><tr>
-                    <th key={-1}>Attribute</th>
-                    { headerCells }
-                </tr></thead>
-                <tbody>{ rows }</tbody>
-            </Table>
+            <div>
+                <div>
+                    <h4 className="pull-left">Samples</h4>
+                    <TableExportButtons className="pull-right" tableData={tableData} />
```

Comment:
These export buttons probably belong inside the table since they operate on the same data source.  But that would require some work on the reactable library.  For now, we'll keep them separate